### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by andrewsanchez**

### CODE REVIEW

File: _includes/links.md [hunk 1]
```diff
@@ -29,7 +29,8 @@
 [rubygems]: https://rubygems.org/pages/download/
 [styles]: https://github.com/swcarpentry/styles/
 [training]: http://swcarpentry.github.io/instructor-training/
-[workshop-repo]: {{ site.workshop_repo }}
+[workshop-repo]: https://github.com/swcarpentry/workshop-template
```
Comment (rgaiacs) [inline _includes/links.md]: This line is correct. Please see https://github.com/swcarpentry/lesson-example/blob/gh-pages/_config.yml#L28.

File: _includes/links.md [hunk 2]
```diff
@@ -29,7 +29,8 @@
...
 [yaml]: http://yaml.org/
 [coc]: https://software-carpentry.org/conduct/
 [coc-reporting]: https://software-carpentry.org/CoC-reporting/
+[lesson-example]: https://swcarpentry.github.io/lesson-example/
```
Comment (rgaiacs) [inline _includes/links.md]: Please use `{{ site.lesson-example }}`. See https://github.com/swcarpentry/lesson-example/blob/gh-pages/_config.yml#L26.

File: setup.md [hunk 3]
```diff
@@ -4,13 +4,13 @@ title: Setup
 root: .
 ---
 
-Our lesson template is kept in the `swcarpentry/styles` repository. The `styles` repository is carefully curated so that 
+Our lesson template is kept in the [swcarpentry/styles repository][styles]. The `styles` repository is carefully curated so that
```
Comment (rgaiacs) [inline setup.md]: Does

~~~
[`swcarpentry/styles`][styles]
~~~

works?

**Comment by andrewsanchez** [comment]
Thanks, still learning about this system (markdown and yml).

**Comment by andrewsanchez** [comment]
When I tried to push a new commit, git wouldn't let me because I was supposedly behind this branch.  This makes no sense to me since I'm the one that pushed the latest commits to this branch.  So I forced my last git push.  Any suggestions to avoid doing this in the future so don't overwrite the commit history?  Am I right in assuming that *not* forcing a git push is preferrable?

**Comment by rgaiacs** [comment]
> When I tried to push a new commit, git wouldn't let me because I was supposedly behind this branch.

Looks like that you amend the commit. From Git (commit) documentation

~~~
--amend
           Replace the tip of the current branch by creating a new commit. The
           recorded tree is prepared as usual (including the effect of the -i
           and -o options and explicit pathspec), and the message from the
           original commit is used as the starting point, instead of an empty
           message, when no other message is specified from the command line
           via options such as -m, -F, -c, etc. The new commit has the same
           parents and author as the current one (the --reset-author option
           can countermand this).
~~~

> This makes no sense to me since I'm the one that pushed the latest commits to this branch.

When you amend this kind of make sense.

> So I forced my last git push.

This is what you need to do if you amend a commit.

> Any suggestions to avoid doing this in the future so don't overwrite the commit history?

Sometimes is OK to overwrite the commit history because it will make it clear. Some projects do it. If you need to access one commit that you amend, you can use `git reflog`. Please see the documentation since it will explain better than me.

**Comment by rgaiacs** [comment]
@andrewsanchez Thanks for the pull request. I'm closing this due #178. Sorry to take over your pull request without asking permission. Because the change on `_includes/links.md` need to be incorporate into styles, I decided to take it over because otherwise this pull request was going to take a little more time before be merge. We are working to streamline this.

**Comment by andrewsanchez** [comment]
Hey @rgaiacs  No worries at all!  Thanks for the comments :)

**Pull request closed without merging**